### PR TITLE
Services1

### PR DESCRIPTION
**Pull request by Justintime50**
# Description

Adds the following services and models:

- ApiKey
- Batch
- Billing
- Brand
- CarrierAccount
- CustomsInfo
- CustomsItem

Updates various base service functions to handle new flows or differing names.

# Testing

Updated tests

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: easypost/services/base_service.py [hunk 1]
```diff
@@ -26,11 +26,11 @@ def _snakecase_name(self, class_name) -> str:
 
     def _class_url(self, class_name) -> str:
```
Comment (nwithan8) [inline easypost/services/base_service.py:27]: This method continues to be the source of my nightmares 😉

**Review by nwithan8 — APPROVED** [review]

File: easypost/services/base_service.py (hunk 1 above)
Comment (Justintime50) [reply easypost/services/base_service.py:27]: If this is your nightmares, you'll do just fine 😉 It's only worked fine for a decade now in multiple libs.

File: easypost/services/batch_service.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+)
+from easypost.services.base_service import BaseService
+
+
+class BatchService(BaseService):
+    def __init__(self, client):
+        self._client = client
+        self._model_class = Batch.__name__
+
+    def create(self, **params) -> List[Any]:
```
Comment (jchen293) [inline easypost/services/batch_service.py:20]:
```suggestion
    def create(self, **params) -> Batch:
```

File: easypost/services/batch_service.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        """Create a Batch."""
+        return self._create_resource(self._model_class, **params)
+
+    def all(self, **params) -> List[Any]:
```
Comment (jchen293) [inline easypost/services/batch_service.py:24]:
```suggestion
    def all(self, **params) -> List[Batch]:
```

File: easypost/services/carrier_account_service.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    def retrieve(self, id) -> CarrierAccount:
+        """Retrieve a CarrierAccount."""
+        return self._retrieve_resource(self._model_class, id)
+
+    def update(self, id, **params) -> CarrierAccount:
+        """Update a CarrierAccount."""
+        return self._update_resource(self._model_class, id, **params)
+
+    def delete(self, id):
```
Comment (jchen293) [inline easypost/services/carrier_account_service.py:47]:
```suggestion
    def delete(self, id) -> None:
```

**Review by jchen293 — APPROVED** [review]
lgtm, left non-blocking feedback

**Comment by Justintime50** [comment]
@jchen293 thanks Jack! I'll make sure to do a good pass over everything once done to fix type hints

**Pull request merged**